src/app/admin/get-started/get-started.component.scss

```diff
@@ -70,7 +72,7 @@ a:focus {
 }
 
 .navbar {
-  padding: 15px 10px;
+  padding: px 10px;
```

Comment:
@suvarnakale - only px? if not required then set 0 only

Reply:
okay mam